### DISCUSSION TITLE
Missing export for form event code

### DISCUSSION THREAD

**Discussion by samsquared** [joyfullservice/msaccess-vcs-addin#372] [Q&A]
I've successfully installed the add-in (works nicely, thank you) and did an export.  Everything seems to be there except code for my form events.  These appear in VBA under my database, "Microsoft Access Class Objects" with separate modules for each form containing event code with the name Form_myformname.  General VBA code appearing under "Modules" was exported.  Is there another option I need to check?

Thanks.

**Comment by joyfullservice** [comment]
No, it shouldn't require any special options. Right now the VBA code for your forms is located at the bottom of the form definition. We are likely changing this in version 4 of the add-in to split the VBA code into a separate file to make a clear distinction between the form layout and the underlying VBA code. See #378 for more details on this proposed change.
